changelog/issue-8068.md

```diff
@@ -0,0 +1,5 @@
+audience: general
+level: minor
+reference: issue 8068
+---
+Client (python): adds python 3.9 support back.
```

Comment:
maybe it's useful to include here why we are doing here? 
"because taskgraph/gecko still uses it smth smth"